jupyterhub/singleuser.py

```diff
@@ -177,7 +181,7 @@ def start(self):
                 self._clear_cookie_cache,
                 self.cookie_cache_lifetime * 1e3,
             ).start()
-        super().start()
+        super(SingleUserNotebookApp, self).start()
```

Comment:
There are multiple calls to old-style super already in this file.  Do you want to mark all of them?  When the time comes to officially drop support, it'll probably be easier to just grep for `super(` than it will be to grep for PY2 comments.